plans/centos7.fmf

```diff
@@ -27,3 +19,33 @@ prepare:
 
 execute:
     how: tmt
+
+/test-ci:
+    adjust:
+      - because: "Plan to be ran when either executed locally, or executed by CI system."
+        when: >-
+          trigger is defined
+          and trigger == code
+        enabled: false
+    discover:
+        how: shell
+        tests:
+        - name: Run CentOS7 tests on CentOS-7, depends on TEST_NAME
+          framework: shell
+          test: cd /root/$REPO_NAME && make $TEST_NAME TARGET=$OS SINGLE_VERSION=$SINGLE_VERSION
+          duration: 3h
+
+/test-plan:
+    adjust:
+      - because: "Plan to be ran when either executed locally, or executed by CI system."
```

Comment:
Could the reason why we have 2 discover sections be described here?
Now the description does not say much.

Reply:
Fixed by commit https://github.com/sclorg/sclorg-testing-farm/pull/49/commits/8c907109cb71797abb70ec1bdce20504971a73d7